testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -942,61 +926,61 @@ def test_copy(self, func, ts):
         ts = u.trajectory.ts
         func(self, self.name, ts)
 
-    def test_copy_slice(self):
-        for p, v, f in itertools.product([True, False], repeat=3):
-            if not any([p, v, f]):
-                continue
-            ts = self._from_coords(p, v, f)
-            yield self._check_copy, self.name, ts
-            yield self._check_independent, self.name, ts
-            yield self._check_copy_slice_indices, self.name, ts
-            yield self._check_copy_slice_slice, self.name, ts
-
-    def _check_bad_slice(self, p, v, f):
-        ts = self._from_coords(p, v, f)
+    @pytest.fixture(params=filter(any,
+                                  itertools.product([True, False], repeat=3)))
+    def some_ts(self, request):
```

Comment:
I don't have a  better name for this fixture. Also this can only be a function scope fixture.